tests/api/test_permissions.py

```diff
@@ -16,69 +16,219 @@
-                   biome=_biome)
-        baker.make("emgapi.Study", pk=112, secondary_accession="SRP0112",
-                   is_public=1, submission_account_id='Webin-000',
-                   biome=_biome)
+        webin_zero_pub_one = baker.make(
+            "emgapi.Study",
+            pk=111,
+            secondary_accession="SRP0111",
+            is_public=1,
+            submission_account_id="Webin-000",
+            biome=biome,
+        )
+        webin_zero_pub_two = baker.make(
+            "emgapi.Study",
+            pk=112,
+            secondary_accession="SRP0112",
+            is_public=1,
+            submission_account_id="Webin-000",
+            biome=biome,
+        )
         # Webin-000 private
-        baker.make("emgapi.Study", pk=113, secondary_accession="SRP0113",
-                   is_public=0, submission_account_id='Webin-000',
-                   biome=_biome)
+        webin_zero_priv = baker.make(
+            "emgapi.Study",
+            pk=113,
+            secondary_accession="SRP0113",
+            is_public=0,
+            submission_account_id="Webin-000",
+            biome=biome,
+        )
 
         # Webin-111 public
-        baker.make("emgapi.Study", pk=114, secondary_accession="SRP0114",
-                   is_public=1, submission_account_id='Webin-111',
-                   biome=_biome)
+        webin_one_pub = baker.make(
+            "emgapi.Study",
+            pk=114,
+            secondary_accession="SRP0114",
+            is_public=1,
+            submission_account_id="Webin-111",
+            biome=biome,
+        )
         # Webin-111 private
-        baker.make("emgapi.Study", pk=115, secondary_accession="SRP0115",
-                   is_public=0, submission_account_id='Webin-111',
-                   biome=_biome)
+        webin_one_priv = baker.make(
+            "emgapi.Study",
+            pk=115,
+            secondary_accession="SRP0115",
+            is_public=0,
+            submission_account_id="Webin-111",
+            biome=biome,
+        )
 
         # unknown public
-        baker.make("emgapi.Study", pk=120, secondary_accession="SRP0120",
-                   is_public=1, submission_account_id=None, biome=_biome)
+        baker.make(
+            "emgapi.Study",
+            pk=120,
+            secondary_accession="SRP0120",
+            is_public=1,
+            submission_account_id=None,
+            biome=biome,
+        )
         # unknown private
-        baker.make("emgapi.Study", pk=121, secondary_accession="SRP0121",
-                   is_public=0, submission_account_id=None, biome=_biome)
+        baker.make(
+            "emgapi.Study",
+            pk=121,
+            secondary_accession="SRP0121",
+            is_public=0,
+            submission_account_id=None,
+            biome=biome,
+        )
+
+        # public run (needeed for auth filtering)
+        public_status = baker.make(
+            "emgapi.Status",
+            pk=4,
```

Comment:
Maybe instead of 4 use `emg_models.Status.PUBLIC`. And similar for other cases where you are instantiating the classes that you added a set of constant in the model

Reply:
Good catch, I forgot to update those values.